### ISSUE TITLE
Skip cookie rows

### ISSUE THREAD

**Issue by unhammer** [tbanel/orgaggregate#16]
Hey, thanks for making orgtbl-aggregate – it's a great addition to org-mode that will mostly just DWIM :-) I just have one minor request:

```
#+NAME: original
|    Day    | Color | Level | Quantity |
|-----------+-------+-------+----------|
|    <c>    |       |       |          |
|  Monday   | Red   |    30 |       11 |
|  Monday   | Blue  |    25 |        3 |
|  Tuesday  | Red   |    51 |       12 |
|  Tuesday  | Red   |    45 |       15 |
|  Tuesday  | Blue  |    33 |       18 |
| Wednesday | Red   |    27 |       23 |
| Wednesday | Blue  |    12 |       16 |
| Wednesday | Blue  |    15 |       15 |
| Thursday  | Red   |    39 |       24 |
| Thursday  | Red   |    41 |       29 |
| Thursday  | Red   |    49 |       30 |
|  Friday   | Blue  |     7 |        5 |
|  Friday   | Blue  |     6 |        8 |
|  Friday   | Blue  |    11 |        9 |

#+BEGIN: aggregate :table "original" :cols "Color count()"
| Color | count() |
|-------+---------|
|       |       1 |
| Red   |       7 |
| Blue  |       7 |
#+END:
```
that empty 1 in the aggregate shouldn't be there

**Comment by tbanel** [comment]
You are right. This is an unforeseen case. I will fix it.

In the meantime, you can move the cookie in the table's header. It will be ignored.

```
#+NAME: original
|    Day    | Color | Level | Quantity |
|    <c>    |       |       |          |
|-----------+-------+-------+----------|
|  Monday   | Red   |    30 |       11 |
|  Monday   | Blue  |    25 |        3 |
|  Tuesday  | Red   |    51 |       12 |
|  Tuesday  | Red   |    45 |       15 |
...
```
_"mostly just DWIM"_, what a nice compliment!
Thanks for reporting. Stay tuned!

**Comment by tbanel** [comment]
Fixed.
Now cookies are ignored in data rows.
They are kept in the header.
Moreover, now multiple rows source headers are propagated as much as possible to the aggregated header.
Details may be found in the README.org.
I added your name to the list of contributors.
I close this issue.
Don't hesitate to reopen it or open a new one if something needs improvement.
The fix is available right now on GitHub and will be in a few hours on Melpa.
Thank you, have fun!
